website/docs/enterprise/releases.mdx

```diff
@@ -7,6 +7,83 @@ import TierLabel from "../_components/TierLabel";
 
 # Releases <TierLabel tiers="enterprise" />
 
+## v0.11.0
+2022-11-25
+
+### Dependency versions
+
+- weave-gitops v0.11.0
+- cluster-controller v1.4.1
+- cluster-bootstrap-controller v0.3.0
+- pipeline-controller v0.0.11
```

Comment:
shouldnt this be optional?

Reply:
Yes it is, only reason I didn't mention it, is because it's deployed by default. Would you prefer to mark it as optional?

Reply:
I'd prefer optional. I can always change that later though

Reply:
Updated